contacts/models.py

```diff
@@ -0,0 +1,16 @@
+from email.mime import message
+from xmlrpc.client import DateTime
+import black
+from django.db import models
+from django.forms import CharField
+from datetime import datetime
+from phonenumber_field.modelfields import PhoneNumberField
+
+class Contact(models.Model):
+    name = models.CharField(max_length=128)
+    email = models.CharField(max_length=64)
+    phone = PhoneNumberField(null=True, blank = True)
+    message = models.TextField(blank=True)
+    date_created = models.DateTimeField(default=datetime.now)
```

Comment:
date_created: types.DateTimeField = models.DateTimeField(_("Date created"), auto_now_add=True, db_index=True)
    date_updated: types.DateTimeField = models.DateTimeField(_("Date updated"), auto_now=True, db_index=True)

Reply:
![error](https://user-images.githubusercontent.com/69903665/196000285-4b84f504-a63c-42a4-8391-d66b3d3b0b33.jpg)

I get this error. it works in regulations, so why do i get this?